inst/examples/dashboard/header.R

```diff
@@ -41,23 +41,112 @@ farItems <- list(
     ariaLabel = "Grid view",
     iconOnly = TRUE,
     iconProps = list(iconName = "Tiles")
-  ),
-  list(
-    key = "info",
-    id = "aboutInfo",
-    href = "#!/about",
-    text = "Info",
-    ariaLabel = "Info",
-    iconOnly = TRUE,
-    iconProps = list(iconName = "Info")
   )
 )
 
-command_bar <- CommandBar(items = items,
+command_bar <- CommandBar(
+  items = items,
   farItems = farItems,
-  style = list(width = "100%"))
+  style = list(width = "100%")
+)
+
+
+appsilon_header <- function() {
+
+  appsilon_breakpoints <- breakpointSystem(
+    "appsilon-breakpoints",
```

Comment:
I'd suggest to forgo "appsilon" from naming here; it makes the names longer but doesn't convey useful information.

```suggestion
header <- function() {

  breakpoints <- breakpointSystem(
    "breakpoints",
```

Reply:
The `appsilon` prefix is still there; was it a conscious decision or accidental omission? :slightly_smiling_face:

Reply:
It was an accidental omission 🙂